### DISCUSSION TITLE
1-wire network (DS9490R) with FUXA?

### DISCUSSION THREAD

**Discussion by mrHubdoc** [frangoteam/FUXA#945] [Q&A]
Is it possible to connect a 1-wire network with a DS9490R Interface Bridge, with temperature sensor ds18b20 to FUXA?
[https://se.farnell.com/analog-devices/ds9490r/adaptor-usb-to-1-wire-rj11/dp/2519165](url)

**Comment by unocelli** [comment]
Hi,
I don't think so and anyway you should find a library for nodejs

**Reply by mrHubdoc** [reply]
-
